### PR TITLE
New url module

### PR DESCRIPTION
**Pull request by tswfi**
Spams the channel with title and or meta description tags when the bot sees a url

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,24 +137,41 @@ Prefix with selected service, for example "!ig add accountname" or "!twitter lis
...
+
+* !url status       - show current status
+* !url title        - spam titles to room
+* !url description  - spam descriptions
+* !url both         - spam both title and description
+* !url off          - stop spamming
+
+Example:
+
+* !urltitles status
```
Comment (vranki) [inline README.md]: Is this !url or !urltitles?

File: README.md [hunk 2]
```diff
@@ -137,24 +137,41 @@ Prefix with selected service, for example "!ig add accountname" or "!twitter lis
...
-without any authentication or api key. 
+without any authentication or api key.
 
 See: https://github.com/taspinar/twitterscraper/tree/master/twitterscraper
 
+#### Url
+
+Watches all messages in a room and if a url is found tries to fetch it and
+spit out the title if found.
+
```
Comment (vranki) [inline README.md]: Is this on by default or does it need to be disabled? Please tell here.

Comment (tswfi) [reply README.md]: Added note that this is disabled by default and needs to be activated for every room this is wanted.

File: README.md (hunk 1 above)
Comment (tswfi) [reply README.md]: Started its life as !urltitles but then descriptions were added and I renamed it but forgot this :) Fixed now.

**Pull request merged**